packages/grafana-data/src/valueFormats/dateTimeFormatters.test.ts

```diff
@@ -196,13 +196,13 @@ describe('duration', () => {
   it('too many decimals', () => {
     const str = toDuration(38898367008, 20, Interval.Millisecond);
     expect(formattedValueToString(str)).toBe(
-      '1 year, 2 months, 3 weeks, 4 days, 5 hours, 6 minutes, 7 seconds, 8 milliseconds'
+      '1 year, 2 months, 3 weeks, 4 days, 5 hours, 6 minutes, 7 seconds, 8 milliseconds, 0 microseconds, 0 nanoseconds'
```

Comment:
Would be curious for thoughts on this. Presumably since there is support for microseconds and nanoseconds we would expect this to return zeros since this goes beyond nanosecond precision but that may not be the case.